tests/tests_interface_pci.yml

```
@@ -7,32 +7,46 @@
     - linux-system-roles.firewall
 
   tasks:
-    - name: Get backend from dbus
-      command: >-
-        dbus-send --system --print-reply --type=method_call
-        --dest=org.fedoraproject.FirewallD1
-        /org/fedoraproject/FirewallD1/config
-        org.freedesktop.DBus.Properties.Get
-        string:"org.fedoraproject.FirewallD1.config"
-        string:"FirewallBackend"
-      ignore_errors: true  # noqa ignore-errors
-      register: result
-      changed_when: false
-
-    - name: Get backend from result
+    - name: Set expected backend
       set_fact:
         nftables_backend:
-          "{{ result is not failed and 'nftables' in result.stdout }}"
+          "{{ true
+            if ansible_distribution in ['RedHat', 'CentOS', 'Fedora']
+            and ansible_distribution_major_version is version('8', '>=')
+            else false }}"
+
+    - name: Find ethernet interface
+      shell: |
+        set -euo pipefail
+        I=$(find /sys/class/net -name 'e*' | head -n1)
+        # containers only have virtual devices; for PCI we need a real one
+        if [ -n "$I" ] && [ -e "$I/device/vendor" ]; then
+          echo "$I"
+        fi
+      register: find_iface
+      changed_when: false
 
     - name: Test interfaces with PCI ids
+      # this can't be tested in containers or similar envs without any real
+      # ethernet devices
+      when: find_iface.stdout != ""
       block:
+        - name: Determine interface vendor/product ID
+          shell: |
+            set -euo pipefail
+            VID="$(sed 's/^0x//' < {{find_iface.stdout}}/device/vendor)"
```

```suggestion
            VID="$(sed 's/^0x//' < {{ find_iface.stdout | quote }}/device/vendor)"
```
ansible wants a single space after `{{` and a single space before `}}`
quote ensures the value can be safely passed to shell